src/components/calcite-dropdown/readme.md

```diff
@@ -40,43 +40,45 @@ You can combine groups in a single dropdown, with varying selection modes:
+| `alignment`            | `alignment`               | specify the alignment of dropdown, defaults to start                                                                                                                | `"center" \| "end" \| "start"`     | `"start"`   |
+| `disableCloseOnSelect` | `disable-close-on-select` | allow the dropdown to remain open after a selection is made if the selection-mode of the selected item's containing group is "none", the dropdown will alwqys close | `boolean`                          | `false`     |
+| `maxItems`             | `max-items`               | specify the max items to display before showing the scroller, must be greater than 0 \*                                                                             | `number`                           | `0`         |
+| `scale`                | `scale`                   | specify the scale of dropdown, defaults to m                                                                                                                        | `"l" \| "m" \| "s"`                | `"m"`       |
+| `selectedItems`        | --                        | **read-only** The currently selected items                                                                                                                          | `HTMLCalciteDropdownItemElement[]` | `[]`        |
+| `theme`                | `theme`                   | specify the theme of the dropdown, defaults to light                                                                                                                | `"dark" \| "light"`                | `undefined` |
+| `type`                 | `type`                    | specify whether the dropdown is opened by hover or click of the trigger element                                                                                     | `"click" \| "hover"`               | `"click"`   |
+| `width`                | `width`                   | specify the width of dropdown, defaults to m                                                                                                                        | `"l" \| "m" \| "s"`                | `"m"`       |
 
 ## Events
 
-| Event                   | Description                                                  | Type                |
-| ----------------------- | ------------------------------------------------------------ | ------------------- |
-| `calciteDropdownSelect` | fires when a dropdown item has been selected or deselected * | `CustomEvent<void>` |
-
+| Event                   | Description                                                   | Type                |
+| ----------------------- | ------------------------------------------------------------- | ------------------- |
+| `calciteDropdownOpen`   | fires when a dropdown has been opened \*                      | `CustomEvent<void>` |
```

Comment:
Maybe for a follow-up, but it'd be good to have calciteDropdownClose on the dropdown and possibly keep the item one internal.

Reply:
Agreed, will just add in this PR. Preference for internal event name from `calcite-dropdown-item` -> `calcite-dropdown`?

Reply:
`calciteDropdownCloseRequest`?

Should we also move the `calciteDropdownItemRegister` and `calciteDropdownItemKeyEvent` internal as well?

Reply:
> calciteDropdownCloseRequest

👍 

> Should we also move the calciteDropdownItemRegister and calciteDropdownItemKeyEvent internal as well?

I think so. I can't think of any practical uses for these outside of the dropdown's initialization.